src/sonic-config-engine/sonic_device_util.py

```diff
@@ -41,7 +46,9 @@ def get_npu_id_from_name(npu_name):
 
 def get_num_npus():
     platform = get_platform_info(get_machine_info())
-    asic_conf_file_path = os.path.join(SONIC_DEVICE_PATH, platform, 'asic.conf')
+    if not platform:
+        return 1
+    asic_conf_file_path = os.path.join(SONIC_DEVICE_PATH, platform, ASIC_CONF_FILENAME)
```

Comment:
I notice we use two nomencaltures interchangably: "ASIC" and "NPU". I feel like we should choose one and stick with it throughout. Thoughts?

Reply:
@jleveque yes, agree we can mark this as To-Do item as this will require change multiple files. We can update this later.

@judyjoseph @arlakshm @SuvarnaMeenakshi